docs/install.rst

```diff
@@ -0,0 +1,35 @@
+Installation
+============
+
+Installation of Cascade
+-----------------------
+Cascade interacts with Dismod-AT underneath. Cascade runs Dismod-AT within
+the IHME infrastructure. Clone it from
+`Cascade on Github <https://github.com/ihmeuw/cascade>`_. Then
+use ``python setup.py install`` to install it into the current
+Python environment or virtual environment.
+
+
+
+Installation of Dismod-AT
+-------------------------
+This describes how to run Dismod-AT on your local computer, not on the
+cluster. It uses Docker, which you have to install separately.
+
+Download and run from the IHME Docker registry. This requires
+VPN access. Maybe you can pull images without the login?::
+
+    docker login reg.ihme.washington.edu
+    docker pull reg.ihme.washington.edu/dismod/dismod_at:0.0.6
```

Comment:
This doesn't work by itself either.

nhawkins@nh-tp:~$ docker pull reg.ihme.washington.edu/dismod/dismod_at:0.0.6

Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post http://%2Fvar%2Frun%2Fdocker.sock/v1.27/images/create?fromImage=reg.ihme.washington.edu%2Fdismod%2Fdismod_at&tag=0.0.6: dial unix /var/run/docker.sock: connect: permission denied